docs/readthedocs/source/doc/LLM/Quickstart/benchmark_quickstart.md

```diff
@@ -0,0 +1,66 @@
+# Benchmarking
+
+We can do benchmarking on Intel GPUs and CPUs. You can refer to [this](https://bigdl.readthedocs.io/en/latest/doc/LLM/Overview/install.html) to prepare the environment.
+
+## Clone Repo
+
+Navigate to your local workspace and then download BigDL from GitHub.
+
+```
+cd your/local/workspace/
+git clone https://github.com/intel-analytics/BigDL.git
+```
+
+## Configure YAML File
+
+Replace the config.yaml under all-in-one folder with your own YAML file. The format of the content is as follows.
```

Comment:
Modify the `config.yaml` under `all-in-one` folder for your own benchmark configurations. The contents of the yaml file are as follows: